dashboard/test/models/blockly_test.rb

```diff
@@ -143,6 +143,23 @@ def assert_equal_xml(a, b)
     assert_equal xml, xml2
   end
 
+  def create_test_data(name, pool, block_text, args, category: 'Events')
```

Comment:
this should have a more-specific name; "test data" is extremely vague, particularly in a test file that contains such a variety of tests.